### PR TITLE
Fix missing mailboxes on new account

### LINKED ISSUE #8041: only a number of mailboxes load when adding a new account

### PR DESCRIPTION
**Pull request by hamza221**
fixes #8041

### CODE REVIEW

File: lib/Controller/AccountsController.php [hunk 1]
```diff
@@ -365,8 +369,10 @@ public function create(string $accountName,
 			);
 		}
 		try {
+			$account = $this->setup->createNewAccount($accountName, $emailAddress, $imapHost, $imapPort, $imapSslMode, $imapUser, $imapPassword, $smtpHost, $smtpPort, $smtpSslMode, $smtpUser, $smtpPassword, $this->currentUserId, $authMethod);
+			$this->mailboxSync->sync($account, $this->logger);
```
Comment (ChristophWurst) [inline lib/Controller/AccountsController.php:373]: suggestion: add try-catch and carry on regardless of errors. mailbox sync should not block the creation of an account

File: lib/Controller/AccountsController.php [hunk 2]
```diff
@@ -384,6 +386,16 @@ public function create(string $accountName,
 			]);
 			return MailJsonResponse::error('Could not create account');
 		}
+		try {
+			$this->mailboxSync->sync($account, $this->logger);
+		} catch (ServiceException $e) {
+			$this->logger->error('Failed syncing the newly created accoutnt' . $e->getMessage(), [
```
Comment (ChristophWurst) [inline lib/Controller/AccountsController.php:392]: typo

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Comment by ChristophWurst** [comment]
To ease incremental reviews please fixup+push instead of amend+force-push until approval and green CI

**Review by ChristophWurst — APPROVED** [review]

**Comment by hamza221** [comment]
Should we ignore codecov?

**Pull request merged**